miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/PoolDto.java

```diff
@@ -42,6 +42,7 @@
   private int dilutionCount;
   private Double insertSize;
   private boolean hasEmptySequence;
+  private boolean hasConsentRevokedLibraries;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "hasConsentRevokedLibraries" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)